packages/cozy-procedures/src/components/documents/DocumentHolder.jsx

```diff
@@ -41,7 +41,7 @@ class DocumentHolder extends Component {
             }
           />
         )}
-        <Chip variant="dashed" className="u-w-100">
+        <Card className="u-flex u-flex-row u-flex-items-center">
```

Comment:
So simple :)

Reply:
I guess it's subjective but I really don't find the declaration that complicated. It's not like they were vastly different css rules, they are all flexbox-related and the names are pretty explicit.